doc/relevancy.md

```diff
@@ -0,0 +1,442 @@
+# Relevancy ranking
+
+This document explains how documents are ranked when you perform a search
+with results sorted by relevancy.
+
+## Stopwords
+
+The stopwords themselves seem to be served by elasticsearch. There's nothing on
+the Search API repo that looks like a list of omitted terms.
+
+There has been some talk of not including stopwords in the past
+https://github.com/alphagov/search-api/commit/0fe6e526c78e8b115371855a91d4b39ccb22098a
+during this time a no_stop method was implemented that seems to stop stopwords
+from being used. Again this seems to be served by ES because it's not defined
+on rummager.
+
+Given that we're taking a stock list of stopwords from ES, we may be missing
+out on a chance to edit them in a way that might be more useful for users.
+I'm not sure how we'd go about changing this but i'd guess that we'd need to
+look into what words could be useful to omit / not omit.
+
+If you search Search API for stopwords then you get literally nothing
+http://search-api.dev.gov.uk/search.json?q=the+that+and+if
+
+If you search for nothing then you get the some of the more frequently clicked links
+http://search-api.dev.gov.uk/search.json?q=
+I'm not sure how useful this is but I thought it was interesting
+
+## Synonyms
+
+A synonym is a word or phrase that means exactly or nearly the same as another
+word or phrase in the same language, for example shut is a synonym of close.
+
+Synonyms are defined in the [synonyms.yml file](https://github.com/alphagov/rummager/blob/master/config/schema/synonyms.yml).
+
+Each synonym group is represented as comma separated lists of synonyms, and
+optionally a "=>" symbol.
+
+### Grouping of synonyms
```

Comment:
This subsection needs rewriting as the non-`=>` synonyms didn't work when we switched to ES5 (for unknown reasons).  There's a commit message somewhere with more detail.